oracle-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -55,3 +55,48 @@ longVarcharColumns=(ID VARCHAR2(100) PRIMARY KEY, COL1 LONG VARCHAR)
 longVarcharColumnsList=(ID,COL1)
 longVarcharValues=VALUES ('User1','48692054686572652120486F772061726520796F75206665656C696E67AbCdEF646179203F')
 outputDatatypesSchema4=[{"key":"ID","value":"string"},{"key":"COL1","value":"string"}]
+
+#ORACLE Invalid Properties
+invalidRef=invalidRef&^*&&*
+oracleInvalidTable=invalidTable
+invalidDatabaseName=invalidDB%$^%*
+invalidImportQuery=select
+invalidTableNameImportQuery=select * from abc;
+invalidDriverName=abcdriver
+invalidHost=testhost
+invalidPort=testPort
+invalidUserName=testUser
+invalidPassword=testPassword
+invalidBoundingQuery=SELECT MIN(id),MAX(id) FROM table
+invalidTable=table
+
+#ORACLE Valid Properties
+connectionArgumentsList=[{"key":"queryTimeout","value":"-1"}]
+batchValue=40
+rowPrefetch=60
+numberOfSplits=2
+zeroValue=0
+splitByColumn=ID
+oracleImportQuery=select * from HR.SOURCETABLE_SANYNMSPDZ where $CONDITIONS
+
+#bq properties
+projectId=cdf-athena
+dataset=test_automation
+
+#bq macro properties
+bqTruncateTable=true
+bqUpdateTableSchema=true
+
+#ORACLE Datatypes
```

Comment:
Why are we creating new datatype columns here, ideally it should be part of the original "datatypeColumns" defined above ?

Reply:
This was created to use with BigQuery sink, which imports table records containing multiple datatypes supported in BigQuery.

Reply:
In that case please rename these variables to a more readable names such as bigqueryColumns.

Reply:
Change done.